### PR TITLE
Replace opties application with 'its over' message

### PR DESCRIPTION
**Pull request by stewsmith**

### CODE REVIEW

File: website-guts/assets/css/pages/opticon-opties.scss [hunk 1]
```diff
@@ -1,88 +1,25 @@
...
-      text-align: left;
-      margin-left: 50px;
-      .description-title { text-align: left; }
-    }
-    .award-categories { margin: 0 10px; }
-    .category-item {
-      width: 50%;
-      padding: 50px 50px 0 0;
+  .message-container {
+    margin: 10vh auto;
```
Comment (tomgenoni) [inline website-guts/assets/css/pages/opticon-opties.scss]: Are `vh` units supported? Seems like there are lots of caveats. http://beta.caniuse.com/#search=vh

**Comment by stewsmith** [comment]
retest this please

File: website-guts/assets/css/pages/opticon-opties.scss (hunk 1 above)
Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-opties.scss]: Looks okay except for IE. IE only has problems with vmax.

**Comment by stewsmith** [comment]
retest this please

**Comment by tomgenoni** [comment]
LGTM

**Pull request merged**